src/components/structures/UserMenu.tsx

```diff
@@ -73,20 +73,29 @@ const CustomStatusSection = () => {
     });
 
     let details: JSX.Element;
+
+    const setStatusMessage = (e) => {
+        e?.preventDefault();
+        if (value !== setStatus) {
+            cli._unstable_setStatusMessage(value);
+        }
+    };
+
     if (value !== setStatus) {
         details = <>
             <p>{ _t("Your status will be shown to people you have a DM with.") }</p>
 
             <RovingAccessibleButton
-                onClick={() => cli._unstable_setStatusMessage(value)}
+                type={value?"submit":"reset"}
```

Comment:
I don't think this will work as the button is outside the `form` element.
If you want this to work you'll have to use the `form` attribute as explained in https://funwithforms.com/posts/form-attribute/

Reply:
I think the button is inside the form element rendered as 
`{ details }`
So  i think it will work fine and also onClick handler is required on type Iprops so removing that will also cause error , Have tested it on my side , works fine

Reply:
Woops, sorry about that! Read the code wrong that looks about right to me